<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selWathToCheck</class>
 <widget class="QDialog" name="selWathToCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelImg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>181</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCentroids">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>190</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Centroids</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxHorizLR">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>220</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Horizontal LR</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxVertLR">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>220</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertical LR</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>320</y>
     <width>241</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="pbLoadImage">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select image&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>  Select image</string>
      </property>
      <property name="icon">
       <iconset resource="lstImgs.qrc">
        <normaloff>:/new/icons/imagenInte/newProject.jpeg</normaloff>:/new/icons/imagenInte/newProject.jpeg</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pbCentroids">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Evaluate centroids&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>  Draw</string>
      </property>
      <property name="icon">
       <iconset resource="lstImgs.qrc">
        <normaloff>:/new/icons/imagenInte/add-542.png</normaloff>:/new/icons/imagenInte/add-542.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBoxLimits">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>250</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Defined Limits</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLimitsCalculated">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>251</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>CalculatedLimits</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="douubleSpinWave">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>280</y>
     <width>74</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxWavelength">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>283</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Wavelength</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCentroidsCalc">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>190</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>CalcCentroids</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="lstImgs.qrc"/>
 </resources>
 <connections/>
</ui>
